ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml

```diff
@@ -6463,7 +6460,7 @@ periodics:
       secret:
         secretName: result-aggregator
 - agent: kubernetes
-  cluster: build06
+  cluster: build09
```

Comment:
@deepsm007 this is not looking like your PR (https://github.com/openshift/release/pull/60000) right?
In your PR the jobs have a new `capability/vpn: vpn` label, and here they have a different cluster assigned.
Can you PTAL?

Reply:
Can you execute `make jobs`?

Reply:
I already did:
```
make ci-operator-config registry-metadata jobs
```